### ISSUE TITLE
apache-rat-plugin license check for drools

### ISSUE THREAD

**Issue by tkobayas** [apache/incubator-kie#6098]
Running check against incubator-kie-10.0.0-rc1-sources.zip for incubator-kie-drools. (as of Sep 25, 2024)
```
mvn apache-rat:check
```

Reports missing license headers.

**Comment by tkobayas** [comment]
```
Files with unapproved licenses:

drools-docs/supplemental-ui/js/search-ui.js
kie-dmn/kie-dmn-xls2dmn-cli/dependency-reduced-pom.xml
kie-dmn/kie-dmn-ruleset2dmn-parent/kie-dmn-ruleset2dmn-cli/dependency-reduced-pom.xml
drools-drlonyaml-parent/drools-drlonyaml-cli/dependency-reduced-pom.xml
drools-drlonyaml-parent/drools-drlonyaml-model/src/test/resources/smoketests/ruleunit.drl.txt
drools-drlonyaml-parent/drools-drlonyaml-schemagen/dependency-reduced-pom.xml
```

**Comment by LightGuard** [comment]
Are these things we ship with our sources? Actually, most are pom.xml files. The test file is probably fine, I think it breaks if we add the header to it. I don't think we ship the docs code.

**Comment by tkobayas** [comment]
@LightGuard Those files are included in incubator-kie-10.0.0-rc1-sources.zip. Anyway, dependency-reduced-pom.xml is generated code, so we simply exclude it. ruleunit.drl.txt was okay to add license (doesn't break a test), so added. search-ui.js already has Mozilla Public License header
